packages/uma-auth-client/src/hooks/useOAuth.tsx

```diff
@@ -93,6 +101,13 @@ export const useOAuth = create<OAuthState>()(
   ),
 );
 
+const isConnectionValid = (state: OAuthState) => {
+  if (!state.nwcExpiresAt) {
+    return false;
```

Comment:
I feel like if the connection has no expiry, shouldn't it last forever? We do, however, need to check if there's a connection URI.

Reply:
oh yea good point